src/OpenSSL/crypto.py

```diff
@@ -1518,6 +1518,18 @@ def set_flags(self, flags):
         """
         _openssl_assert(_lib.X509_STORE_set_flags(self._store, flags) != 0)
 
+    def set_time(self, vfy_time):
+        """Set verification time to this store.
+
+        .. versionadded: 16.3.0
+
+        :param datetime vfy_time: The verification time to set on this store.
+        :return: ``None`` if the verification time were successfully set.
+        """
+        param = _lib.X509_VERIFY_PARAM_new()
```

Comment:
You'll need to add a `param = _ffi.gc(param, X509_VERIFY_PARAM_free)` to this. `X509_STORE_set1_param` increfs it so without a corresponding free to decref after creation it will leak.